src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -70,14 +58,38 @@ class remove_virtual_functionst
   get_method(const irep_idt &class_id, const irep_idt &component_name) const;
 };
 
-remove_virtual_functionst::remove_virtual_functionst(
-  symbol_table_baset &_symbol_table,
-  const class_hierarchyt &_class_hierarchy)
-  : ns(_symbol_table),
-    symbol_table(_symbol_table),
-    class_hierarchy(_class_hierarchy)
+class remove_virtual_functionst : public get_virtual_calleest
```

Comment:
Is "is-a" really the correct relationship? My guess would have been that `remove_virtual_functionst` uses-a/has-a `get_virtual_calleest`?!

Reply:
Sure, was really just minimising churn. Changed to use-a.